server/repository/src/migrations/v2_09_00/resync_existing_vaccination_records.rs

```diff
@@ -0,0 +1,21 @@
+use crate::migrations::*;
+
+pub(crate) struct Migrate;
+
+impl MigrationFragment for Migrate {
+    fn identifier(&self) -> &'static str {
+        "resync_existing_vaccination_records"
+    }
+
+    fn migrate(&self, connection: &StorageConnection) -> anyhow::Result<()> {
+        sql!(
+            connection,
+            r#"
+                INSERT INTO changelog (table_name, record_id, row_action, name_link_id, store_id, is_sync_update, source_site_id)
+                SELECT table_name, record_id, row_action, name_link_id, store_id, is_sync_update, source_site_id FROM changelog WHERE table_name = 'vaccination' and source_site_id IS NOT NULL; 
```

Comment:
Copying change log gets the correct site id, rather than the approach for vaccine_course like this...
```
                INSERT INTO changelog (table_name, record_id, row_action)
                SELECT 'vaccine_course', id, 'UPSERT' FROM vaccine_course; 
```
I think these are still fine, but just the remote records need to include the right source site...